pypeit/coadd2d.py

```diff
@@ -256,7 +286,7 @@ def coadd(self, only_slits=None, interp_dspat=True):
                 slits.mask, flag=slits.bitmask.exclude_for_reducing)))
             good_slits = np.where(np.invert(reduce_bpm))[0]
         else:
-            embed(header='DEAL WITH bitmask')
```

Comment:
I think this should read only_slits option is not implanted yet.

Reply:
Updated.